src/es/components/prototypes/Validation.js

```diff
@@ -0,0 +1,379 @@
+// @ts-check
+
+import { Shadow } from './Shadow.js'
+
+export const Validation = (ChosenClass = Shadow()) => class Validation extends ChosenClass {
+  /**
+   * Creates an instance of Shadow. The constructor will be called for every custom element using this class when initially created.
```

Comment:
A bit more description would be appreciated.

Reply:
Added short description of Validation file